src/main/webapp/view/discussion/discussion.js

```diff
@@ -1,24 +0,0 @@
-// Copyright 2020 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import LectureView from '../lecture-view.js';
-
-export const COMMENT_TYPE_REPLY = 'REPLY';
-export const COMMENT_TYPE_QUESTION = 'QUESTION';
-export const COMMENT_TYPE_NOTE = 'NOTE';
```

Comment:
Can we keep this file?  It will contain info on comment types, which is used by all other discussion javascript files.

We could also move them all to discussion-comment.js

Reply:
Sorry I think I misunderstood what you said last week, thought we were removing this file.

Added back.